buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json

```diff
@@ -193,7 +193,25 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -- /p:OverwriteOnPublish=true",
+        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -PublishPackages -- /p:OverwriteOnPublish=true",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Publish symbol packages",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -PublishSymbols -- /p:OverwriteOnPublish=true",
```

Comment:
@weshaggard @karajas do I need to use a new/different Azure account for these packages?

Reply:
For an individual legs you can use the same account but we should start push the packages to a common container instead of a unique one per build. Instead the unique part should become the blob relative path. See https://github.com/dotnet/core-eng/issues/753

Reply:
I'm not sure what the consensus on this was, are we using different or the same account as corefx